Admin/AclAdminExtension.php

```diff
@@ -47,10 +48,17 @@ class AclAdminExtension extends AdminExtension
      * @param array                    $roleHierarchy
      */
     public function __construct(
-        SecurityContextInterface $securityContext,
+        $securityContext,
         Connection $databaseConnection,
         array $roleHierarchy = array()
     ) {
+        if (!($securityContext instanceof SecurityContextInterface) // check type of $securityContext
+                && !($securityContext instanceof TokenInterface)) {
+            throw new InvalidArgumentException('$securityContext must be an instanse of SecurityContextInterface or of TokenInterface '
```

Comment:
`$tokenStorage should be an instance of Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface or Symfony\Component\Security\Core\SecurityContextInterface.`